log_data.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+file_path = 'D:\Python\Repo\Training_Repo\log_data.txt'
+
+def top_downloader(path):
+    with open(file_path) as f:
+        calc_dict = dict()
+        for line in f:
+            ip_date = line.split()[1][:12] + '] ' + line.split()[0]
+            size = int(line.split()[2])
+            if ip_date not in calc_dict:
+                calc_dict[ip_date] = size
+            else:
+                calc_dict[ip_date] = calc_dict[ip_date] + size
+    res_dict = dict()
+    size_dict = dict()
+    for key in calc_dict:
+        date = key.split()[0]
```

Comment:
A more pythonic way to assign here is 
date, ip = key.split()[:1]

https://treyhunner.com/2018/03/tuple-unpacking-improves-python-code-readability/

Reply:
In Python3 you may use

date, ip, *rest = key.split()